source/functions/BEMIO/readCAPYTAINE.m

```diff
@@ -27,7 +30,15 @@
 p = waitbar(0,'Reading Capytaine netcdf output file...'); %Progress bar
 
 hydro(F).code = 'CAPYTAINE';
-[meshdir,name,~] = fileparts(filename);
+
+[base_dir, name, ~] = fileparts(filename);
```

Comment:
Update to camelCase

Reply:
Can do but I think all of the other variables in this file are using underscore convention?

Reply:
Fair point, I forgot that we haven't yet updated the hydro struct variables to camelCase before since that would be a large change. In that case let's leave it as you have it. It can be addressed in a future refactoring if necessary.